schemas/v3.1/schema-base.yaml

```diff
@@ -1,20 +1,20 @@
-$id: 'https://spec.openapis.org/oas/3.1/schema-base/2022-10-07'
+$id: 'https://spec.openapis.org/oas/3.1/schema-base/WORK-IN-PROGRESS'
 $schema: 'https://json-schema.org/draft/2020-12/schema'
 
-description: The description of OpenAPI v3.1.x documents using the OpenAPI JSON Schema dialect, as defined by https://spec.openapis.org/oas/v3.1.0
+description: The description of OpenAPI v3.1.x Documents using the OpenAPI JSON Schema dialect
 
-$ref: 'https://spec.openapis.org/oas/3.1/schema/2022-10-07'
+$ref: 'https://spec.openapis.org/oas/3.1/schema/WORK-IN-PROGRESS'
 properties:
   jsonSchemaDialect:
     $ref: '#/$defs/dialect'
 
 $defs:
   dialect:
-    const: 'https://spec.openapis.org/oas/3.1/dialect/base'
+    const: 'https://spec.openapis.org/oas/3.1/dialect/base/WORK-IN-PROGRESS'
```

Comment:
Is this necessary? See comment on the referenced file.

Reply:
Yes.